index.html

```diff
@@ -0,0 +1,129 @@
+        <li>My page</li>
+        <li>logout</li>
+      </ul>
+    </nav>
+    <div class="second">
+      <img class="logo" src="Assets/mawazine.png" alt="logo">
+      <nav class="nav-bar">
+        <ul>
+          <li><a href="index.html" class="nav-link" id="Home">Home</a></li>
+          <li><a href="about.html" class="nav-link">About</a></li>
+          <li><a href="#Program" class="nav-link">Program</a></li>
+          <li><a href="#Geusts" class="nav-link" >Guests</a></li>
+          <li><a class="nav-link" href="#Parteners">Partners</a></li>
+          <li><a href="#Envents" class="nav-link">Event</a></li>
+          <li><a href="#" class="nav-link"><span id="mawazine">mawazine</span> </a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+    <div class="head">
+      <p class="slogan">"Largest Music Festival in Africa"</p>
+      <h1 class="title">
+        Mawazine Music <br>
+      <span class="title-content">Festival</span> <br>
+      <span class="title-content">2022</span></h1>
+    </div>
+    <div class="content">
+      <p>The Mawazine Festival in Rabat, Morocco is a music festival of incredible scale. It has been named the largest festival in the world with the largest ratio of visitors per stage. The name “Mawazine” roughly translates to Rhythms of the World.</p>
+    </div>
+    <h3 class="event-date">2022.07.25(MON) ~ 27(WED)</h3>
+    <h4 class="event-location">@Agdal Rue 5 | Hilaton | Rabat</h4>
+  </main>
+  <section id="Program" class="main-section">
+   <div class="section-title">
+    <h2 >Main Program</h2>
+    <span class="bottom-title"></span>
+   </div>
+    <div class="cards">
+      <div class="card">
+        <img src="Assets/program_icon_01.png" alt="">
+        <h3>MMF 2022</h3>
+        <p>The Mawazine Festival first started in 2001 as a much smaller event dedicated to World Music. </p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_02.png" alt="">
+        <h3>MUSIC</h3>
+        <p>Appreciate various music creations, from different parts of Africa. Feel the culture from Millenium Hall..</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_03.png" alt="">
+        <h3>TICKETS</h3>
+        <p>Come a get your ticket from one of our agences and do not miss the amizing Show.</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_04.png" alt="">
+        <h3>WORKSHOPS</h3>
+        <p>Along with this big event that will be held in Rabat, there are some wonderful WORKSHOPS that we do not want to miss</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_05.png" alt="">
+        <h3>MMF Ingite</h3>
+        <p>Get opportunities to network with the attendees and hear some professional advice from top artists.</p>
+      </div>
+    </div>
+    <button class="join" type="submit">Join MMF NOW</button>
+  </section>
+  <section class="speakers-sec" id="Geusts">
```

Comment:
Consider reducing the size of speaker images to match them with the original design.
This is how your speaker's section looks now. 
![image](https://user-images.githubusercontent.com/33268927/182562853-9b910ec5-d748-4634-b69d-82870c4a8dab.png)